bundle/manifests/mgc-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -16,7 +16,7 @@ spec:
   - name: https
     port: 8443
     protocol: TCP
-    targetPort: https
+    targetPort: metrics
```

Comment:
If I am not mistaken this is copied to the bundle from https://github.com/Kuadrant/multicluster-gateway-controller/blob/main/config/prometheus/service.yaml

Reply:
can you elaborate a little more please?

Reply:
The definition for this lives in the config directory. This definition is used when we do a `make bundle` to create the bundle